### PR TITLE
Update .NET Backup Overview

### PR DESCRIPTION
**Pull request by JeremyLikness**
Added overview, links to relevant documents, and created an example (verified in VS 2017). @CamSoper will post to team for sign-off.

### CODE REVIEW

**Comment by openpublishbuild** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api/overview/azure/index.yml](https://github.com/JeremyLikness/azure-docs-sdk-dotnet/blob/patch-6/api/overview/azure/index.yml) | :x:Error | | [Details](#user-content-api%2Foverview%2Fazure%2Findex.yml)
[]() | :x:Error | | [Details](#user-content-)

<a name="api%2Foverview%2Fazure%2Findex.yml"></a>
### [api/overview/azure/index.yml](https://github.com/JeremyLikness/azure-docs-sdk-dotnet/blob/patch-6/api/overview/azure/index.yml)
 - **[Error]** Error transforming model &quot;W:\1qjl-o-unknown\raw-\qxovqjz1.mx0\rawmodel\api\overview\azure\index.raw.json&quot; generated from &quot;api/overview/azure/index.yml&quot; using &quot;ManagedReference.html.primary.js&quot;. Error running Transform function inside template preprocessor: 

<a name=""></a>
### []()
 - **[Error]** Error transforming model &quot;W:\1qjl-o-unknown\raw-\qxovqjz1.mx0\rawmodel\api\overview\azure\index.raw.json&quot; generated from &quot;api/overview/azure/index.yml&quot; using &quot;ManagedReference.html.primary.js&quot;. Error running Transform function inside template preprocessor: 

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C24%5C14458122-e42f-10e3-bd3d-ee1c7494cfbf%5CPullRequest%5C201707241657201705-261%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api/overview/azure/backup.md [hunk 1]
```diff
@@ -36,21 +39,18 @@ Install-Package Microsoft.Azure.Management.RecoveryServices.Backup
...
 > [!div class="nextstepaction"]
 > [Explore the management APIs](/dotnet/api/overview/azure/backup/management)
 
-
-## Samples
-
---Samples list here--
-
 [PackageManager]: https://docs.microsoft.com/nuget/tools/package-manager-console
 [DotNetCLI]: https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-add-package
```
Comment (MaximRouiller) [inline api/overview/azure/backup.md]: `en-us` 😄

File: api/overview/azure/backup.md [hunk 2]
```diff
@@ -17,12 +17,15 @@ ms.service: multiple
 
 ## Overview
 
---Overview text here--
+Azure Backup is the cloud service you can use to back up, protect, and restore your data. Azure Backup replaces your existing on-premises or off-site backup solution with a cloud-based solution that is reliable, secure, and cost-competitive. Azure Backup offers multiple components that you download and deploy on the appropriate computer, server, or in the cloud. The component, or agent, that you deploy depends on what you want to protect. All Azure Backup components can be used to back up data to a Recovery Services vault in Azure.
 
+Learn more about Azure Backup by reading [What is Azure Backup?](/azure/backup/backup-introduction-to-azure-backup)
+
+Get started with a walkthrough to [Back up Azure virtual machines to Recovery Services vaults](https://docs.microsoft.com/en-us/azure/backup/backup-azure-vms-first-look-arm).
```
Comment (MaximRouiller) [inline api/overview/azure/backup.md]: `en-us` 😄

**Comment by MaximRouiller** [comment]
#sign-off

**Comment by openpublishbuild** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api/overview/azure/index.yml](https://github.com/JeremyLikness/azure-docs-sdk-dotnet/blob/patch-6/api/overview/azure/index.yml) | :x:Error | | [Details](#user-content-api%2Foverview%2Fazure%2Findex.yml)
[]() | :x:Error | | [Details](#user-content-)

<a name="api%2Foverview%2Fazure%2Findex.yml"></a>
### [api/overview/azure/index.yml](https://github.com/JeremyLikness/azure-docs-sdk-dotnet/blob/patch-6/api/overview/azure/index.yml)
 - **[Error]** Error transforming model &quot;W:\22lo-o-4b44d49a\raw-\zf45kev4.1cu\rawmodel\api\overview\azure\index.raw.json&quot; generated from &quot;api/overview/azure/index.yml&quot; using &quot;ManagedReference.html.primary.js&quot;. Error running Transform function inside template preprocessor: 

<a name=""></a>
### []()
 - **[Error]** Error transforming model &quot;W:\22lo-o-4b44d49a\raw-\zf45kev4.1cu\rawmodel\api\overview\azure\index.raw.json&quot; generated from &quot;api/overview/azure/index.yml&quot; using &quot;ManagedReference.html.primary.js&quot;. Error running Transform function inside template preprocessor: 

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C24%5C14458122-e42f-10e3-bd3d-ee1c7494cfbf%5CPullRequest%5C201707241711434875-261%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**